addons/payment_asiapay/controllers/main.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import pprint
+
+from odoo import _, http
+from odoo.exceptions import ValidationError
+from odoo.http import request
+
+_logger = logging.getLogger(__name__)
+
+class AsiaPayController(http.Controller):
+    _return_url = "/payment/asiapay/return"
+    _notify_url = "/payment/asiapay/notify"
+
+    @http.route(_return_url, type="http", auth="public", methods=['GET'])
+    def asiapay_return(self, **data):
```

Comment:
If the redirection request includes feedback data, why not process them?

Reply:
The redirect data doesn't have any way to confirm it is from Asiapay.

Reply:
I don't get it. Do you mean that AsiaPay doesn't include any useful data?

Reply:
```
received AsiaPay return data:
{'Ref': 'S00038-20220819064057'} 
```
This is the data returned from the _return_url. I guess this is not very useful?